graylog2-server/src/main/java/org/graylog2/datatiering/DeleteSnapshotEvent.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.datatiering;
+
+public record DeleteSnapshotEvent(SearchableSnapshotInfo searchableSnapshotInfo) {
```

Comment:
This event name sounds more like a command than an event. And one thing you might have missed is that we didn't want to use the word "snapshot" for the general things, so "snapshot" is an implementation detail. I think we should use warm index instead. The name for the class could maybe be something like `WarmTierIndexDeletedEvent`.

Reply:
what about WarmIndexDelete? I am against deleted because this means it should be gone but is not the case